react-data-converter/README.md

```diff
@@ -0,0 +1,53 @@
+# Gorilla Zoo Data Converter
+
+## Overview
+
+This project provides a web application for fetching API calls from a API documentation urls into a specified format, to easily generate training points to train a finedtuned LLM model to perform API calls.
```

Comment:
Maybe call it sub-directory instead of "project" so that the organization is clearer?